observermedia/eight-day-week
# Fix: display correct title when creating new section

### Description of the Change
This PR fixes issue found here: https://github.com/10up/eight-day-week/issues/54#issuecomment-693131923

> issue creating Sections within Print Issues, name entered for Section does not save and Template is used by default instead

### Verification Process

1. Upgrade WP to 5.5.1
2. Create a new section.
3. See the section title displays correctly.



### Checklist:



- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/eight-day-week/blob/develop/CONTRIBUTING.md) document.



### Applicable Issues
